src/main.ts

```diff
@@ -104,7 +106,14 @@ async function syncSession () {
   // Auto authenticate user only if they are not trying a login or logout
   if (!isSigningIn() && !isSigningOut()) {
     // Initialize token service which will do a check-sso to initiate session
-    await KeycloakService.initializeToken(null).then(() => {}).catch(err => {
+    await KeycloakService.initializeToken(null).then(() => {
+      const token = ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)
+      if (!token) {
+        // when there is no token in session storage, e.g. because token has expired due to user inactivity,
+        // make sure to clear out account info saved on previous login
+        ConfigHelper.removeFromSession(SessionStorageKeys.CurrentAccount)
+      }
+    }).catch(err => {
```

Comment:
I'm surprised to see this here, since it is bolyachevets' commit from last week and it's [already in main branch](https://github.com/bcgov/namerequest/blob/93f82f43983165d8661024fb50ebe4d56ddd9343/src/main.ts#L109).

Anyway, I think there's no harm being done here.

Reply:
This PR was created before boylachevets's commit. I rebased. Looks like it's still comparing with the version when I created the PR.

Reply:
Yeah, weird. I think I've seen this before in Github side-by-side compares. Anyway, not a problem here. Resolving.